### PR TITLE
Update summary.md

### PR DESCRIPTION
**Pull request by ijkaylin**
Adding description of buttons on Metrics Summary

### What does this PR do?
To describe bulk tag configuration and bulk percentiles buttons on Metrics Summary

### Preview

https://docs-staging.datadoghq.com/kathy.lin/summary/metrics/summary/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by jtappa — CHANGES_REQUESTED** [review]
Can you please add a shadow box around the two buttons in the first image? It's not immediately apparent that's what we're referencing since there's a lot of text in that image.

Thanks!

**Comment by ijkaylin** [comment]
updated image with box

File: content/en/metrics/summary.md [hunk 1]
```diff
@@ -29,6 +29,18 @@ The search bars provide the most comprehensive set of actions to filter the list
...
+## Configuration of multiple metrics 
+There are two buttons that allow you to configure multiple metrics at a time: 
+
+{{< img src="metrics/summary/configurationbuttons.jpg" alt="Bulk Configuration Buttons"  style="width:75%;">}}
+
+* **Calculate Percentiles**: Add percentile aggregations to multiple distribution metrics
+
+{{< img src="metrics/summary/bulkpercentiles.jpg" alt="Bulk Percentiles"  style="width:75%;">}}
+
+* **Configure Tags**: Configure tags on multiple distribution metrics (Metrics without Limits* beta users will be able to use this functionality on counts,gauges,rates as well)
```
Comment (jtappa) [inline content/en/metrics/summary.md:41]:
```suggestion
* **Configure Tags**: Configure tags on multiple distribution metrics (Metrics without Limits* beta users will be able to use this functionality on counts, gauges, and rates as well).
```

File: content/en/metrics/summary.md [hunk 2]
```diff
@@ -29,6 +29,18 @@ The search bars provide the most comprehensive set of actions to filter the list
 
 {{< img src="metrics/summary/facets.jpg" alt="Metrics Facet Panel"  style="width:75%;">}}
 
+## Configuration of multiple metrics 
+There are two buttons that allow you to configure multiple metrics at a time: 
+
+{{< img src="metrics/summary/configurationbuttons.jpg" alt="Bulk Configuration Buttons"  style="width:75%;">}}
+
+* **Calculate Percentiles**: Add percentile aggregations to multiple distribution metrics
```
Comment (jtappa) [inline content/en/metrics/summary.md:37]:
```suggestion
* **Calculate Percentiles**: Add percentile aggregations to multiple distribution metrics.
```

**Review by jtappa — CHANGES_REQUESTED** [review]

**Review by jtappa — APPROVED** [review]

**Review by jtappa — APPROVED** [review]

**Pull request merged**